src/base/ghost_point_neighbors.C

```diff
@@ -67,30 +89,97 @@ void GhostPointNeighbors::operator()
   CouplingMatrix * nullcm = nullptr;
 
   for (const auto & elem : as_range(range_begin, range_end))
+  {
```

Comment:
libMesh indentation is two spaces to the brace, plus another two spaces to the code in the next block.